app/src/main/java/com/example/shivansh/to_dolist/MainActivity.java

```diff
@@ -24,7 +24,7 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onClick(View view) {
                 Intent newtask = new Intent(this, todo.class);
```

Comment:
you can't use directly "this" to pass context in this case, because method is defined in anonymous class. It is giving compilation errors.